docs/features.md

```diff
@@ -354,15 +365,15 @@ Known limitations:
 The following features are supported by the LSP specification but not implemented in HLS.
 Contributions welcome!
 
-| Feature                | Status                                                                                   | [LSP method](./what-is-hls.md#lsp-terminology)      |
-|------------------------|------------------------------------------------------------------------------------------|-----------------------------------------------------|
-| Signature help         | Unimplemented                                                                            | `textDocument/signatureHelp`                        |
-| Jump to declaration    | Unclear if useful                                                                        | `textDocument/declaration`                          |
-| Jump to implementation | Unclear if useful                                                                        | `textDocument/implementation`                       |
-| Folding                | Unimplemented                                                                            | `textDocument/foldingRange`                         |
-| Semantic tokens        | Unimplemented                                                                            | `textDocument/semanticTokens`                       |
-| Linked editing         | Unimplemented                                                                            | `textDocument/linkedEditingRange`                   |
-| Document links         | Unimplemented                                                                            | `textDocument/documentLink`                         |
-| Document color         | Unclear if useful                                                                        | `textDocument/documentColor`                        |
-| Color presentation     | Unclear if useful                                                                        | `textDocument/colorPresentation`                    |
-| Monikers               | Unclear if useful                                                                        | `textDocument/moniker`                              |
+| Feature                | Status            | [LSP method](./what-is-hls.md#lsp-terminology) |
+| ---------------------- | ----------------- | ---------------------------------------------- |
+| Signature help         | Unimplemented     | `textDocument/signatureHelp`                   |
+| Jump to declaration    | Unclear if useful | `textDocument/declaration`                     |
+| Jump to implementation | Unclear if useful | `textDocument/implementation`                  |
+| Folding                | Unimplemented     | `textDocument/foldingRange`                    |
```

Comment:
you implemented it!

Reply:
Should I just remove folding from here? Or should I mention implemented?

Reply:
It's a table of unimplemented stuff, so remove it!